.travis.yml

```diff
@@ -4,6 +4,7 @@ python:
 script:
   - PYTHONPATH=. python tools/test/config_test/config_test.py
   - python tools/build_travis.py
+  - py.test tools/tests/toolchain/api.py
```

Comment:
Can you move this above the `python tools/build_travis.py` script? That script takes a long time, I'd rather know about tool test failures first.